pages/common/tlmgr-gui.md

```diff
@@ -0,0 +1,29 @@
+# tlmgr gui
+
+> Start the graphical user interface of `tlmgr`.
```

Comment:
```suggestion
> Start a graphical user interface of `tlmgr`.
```

Reply:
If you agree with this change, make sure to basically apply it to all of the command descriptions!

Reply:
Hm, I'm not sure. You typically only start one instance at a time.

Reply:
It feels to possessive, does it need the GUI or is there a terminal version I haven't seen due to my ignorance of the command

Reply:
It's mainly a terminal program

Reply:
What do you think of 
```suggestion
> Start a graphical user interface version of `tlmgr`.
```

Reply:
What about this?
```suggestion
> Start a graphical user interface for `tlmgr`.
```

Reply:
Sounds good.